src/components/Menu/Menu.cy.tsx

```diff
@@ -59,9 +59,10 @@ const closeMenuWithKeyboard = (keys: any, hasTrigger = true) => {
 const openWithKeyboard = (hasTrigger = true) => {
     if (hasTrigger) {
         cy.get(MENU_SELECTOR).find('button').eq(0).focus().realPress('Enter');
-    } else {
-        cy.get('body').trigger('keydown', { key: 'Tab' });
     }
+    // else {
+    // cy.get('body').trigger('keydown', { key: 'Tab' });
```

Comment:
This block was supposed to add focus to the menu without a trigger. I saw now it is auto focusing the first element by default because of the keyboard navigation hook.
I was thinking here: maybe this can be an issue if in a page that uses the `Menu` without a trigger, the keyboard navigation will start on that menu first item. 
WDYT about changing the keyboard navigation hook to not autofocus it if no trigger element?
cc: @imoutaharik